### PR TITLE
TSK-1865: Reorder methods in ClassificationService

### PR DESCRIPTION
**Pull request by ensarevlideveloper**
New ordering follows CRUD principles for better readability.
---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [X] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [X] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [X] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

**Comment by ensarevlideveloper** [comment]
_bump_

File: lib/taskana-core/src/main/java/pro/taskana/classification/api/ClassificationService.java [hunk 1]
```diff
@@ -17,6 +17,61 @@
...
+   *     exist in the configuration
+   * @throws MalformedServiceLevelException if the {@linkplain Classification#getServiceLevel()
+   *     serviceLevel} does not comply with the ISO 8601 specification
+   * @throws InvalidArgumentException if the {@linkplain Classification} contains invalid properties
+   */
+  Classification createClassification(Classification classification)
+      throws ClassificationAlreadyExistException, NotAuthorizedException, DomainNotFoundException,
+          InvalidArgumentException, MalformedServiceLevelException;
+
+  // READ
```
Comment (mustaphazorgati) [inline lib/taskana-core/src/main/java/pro/taskana/classification/api/ClassificationService.java:74]: space missing below

File: lib/taskana-core/src/main/java/pro/taskana/classification/api/ClassificationService.java [hunk 2]
```diff
@@ -49,6 +104,29 @@ Classification getClassification(String key, String domain)
...
+   *     Classification#getModified() modified} timestamp differs from the one in the database
+   * @throws MalformedServiceLevelException if the {@linkplain Classification#getServiceLevel()
+   *     serviceLevel} does not comply with the ISO 8601 specification
+   * @throws InvalidArgumentException if the {@linkplain Classification} contains invalid properties
+   */
+  Classification updateClassification(Classification classification)
+      throws ClassificationNotFoundException, NotAuthorizedException, ConcurrencyException,
+          InvalidArgumentException, MalformedServiceLevelException;
+
+  // DELETE
```
Comment (mustaphazorgati) [inline lib/taskana-core/src/main/java/pro/taskana/classification/api/ClassificationService.java:129]: space missing below

**Review by mustaphazorgati — CHANGES_REQUESTED** [review]
The order of the overloaded method's is not consistent (especially not with the already reordered WorkbasketService).

Please use first the methods, which use the technical id, then the method using key + domain

**Review by mustaphazorgati — APPROVED** [review]

**Pull request merged**